ground/gcs/src/plugins/uavsettingsimportexport/uavsettingsimportexportfactory.h

```diff
@@ -36,6 +39,7 @@ class UAVSETTINGSIMPORTEXPORT_EXPORT UAVSettingsImportExportFactory : public QOb
 public:
     UAVSettingsImportExportFactory(QObject *parent = 0);
     ~UAVSettingsImportExportFactory();
+    static bool updateObject(UAVObject *obj, QDomNode * node);
```

Comment:
I don't see this new method being defined anywhere. Can you check if it works with this line (and the one
above on 33) deleted?